e2e/e2etests/e2etests.go

```diff
@@ -472,6 +474,14 @@ var AllE2ETests = []runner.E2ETest{
 		},
 		TestTONDepositAndCall,
 	),
+	runner.NewE2ETest(
+		TestTONDepositRefundName,
+		"deposit TON into ZEVM; expect refund",
```

Comment:
```suggestion
		"deposit TON into ZEVM and call a smart contract that reverts; expect refund",
```
The refund is because of a smart contract that reverts right?